### PR TITLE
Moved about page to separate controller

### LINKED ISSUE #661: Move about page to seprate controller
About page is currently rendered from logix, controller, move to separate controller

https://github.com/CircuitVerse/CircuitVerse/blob/stories%2Fnew_ux/app/controllers/logix_controller.rb#L30

### PR DESCRIPTION
**Pull request by MrBartusek**
Resolve #661
GCI task

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -30,6 +30,8 @@
 
   get '/search', to: "search#search"
 
+  get '/about', to: "about#about"
```
[the remark this answers is not in the document]
Comment (MrBartusek) [reply config/routes.rb]: Search route has double quote so i don't know if i should change this

File: app/controllers/about_controller.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class AboutController < ApplicationController
+  def about
```
[the remark this answers is not in the document]
Comment (MrBartusek) [reply app/controllers/about_controller.rb]: Other empty methods is not on single line so i don't know if i should change this

File: config/routes.rb (hunk 1 above)
Comment (ankitkataria) [inline config/routes.rb]: Could we change this method name to `index` to make it more resource like? 
Also I think you'll have to move `apps/views/logix/about.html.erb` to its appropriate location.

Comment (MrBartusek) [reply config/routes.rb]: Actually `app/views/logix/about.html.erb` has been already moved to `app/views/about/about.html.erb` (now changed to `app/views/about/index.html.erb`)
I just changed method name to `index`  in https://github.com/CircuitVerse/CircuitVerse/pull/728/commits/9907b986a3e8c5a875ca490f1841a96081ddb6f9

File: app/assets/stylesheets/about.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the about controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (tachyons) [inline app/assets/stylesheets/about.scss]: remove empty file

File: app/helpers/about_helper.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module AboutHelper
```
Comment (tachyons) [inline app/helpers/about_helper.rb]: remote empty helper

File: config/routes.rb [hunk 5]
```diff
@@ -30,6 +30,8 @@
 
   get '/search', to: "search#search"
 
+  get '/about', to: "about#index"
```
Comment (tachyons) [inline config/routes.rb]: resource :about, only: index

File: spec/controllers/about_controller_spec.rb [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe AboutController, type: :request do
+  it "should get about page" do
```
[the remark this answers is not in the document]
Comment (MrBartusek) [reply spec/controllers/about_controller_spec.rb]: Other tests are not using single quote so I don't know if I should change this

File: spec/controllers/about_controller_spec.rb [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require "rails_helper"
```
[the remark this answers is not in the document]
Comment (MrBartusek) [reply spec/controllers/about_controller_spec.rb]: Other tests are not using single quote so I don't know if I should change this

File: config/routes.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (alanjsph) [reply config/routes.rb]: Its ok, you can follow the standard guidelines in your PR irrespective of what is already there. (Unless it is a change in design patterns and all)

File: app/controllers/about_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (alanjsph) [reply app/controllers/about_controller.rb]: Yes you should. Please go ahead

File: spec/controllers/about_controller_spec.rb (hunk 6 above)
[the remark this answers is not in the document]
Comment (alanjsph) [reply spec/controllers/about_controller_spec.rb]: Yes, please go ahead and address this in your PR

**Comment by alanjsph** [comment]
Good work @MrBartusek :)

**Pull request merged**